package.json

```diff
@@ -18,6 +18,8 @@
     "featureflags": "yarn --cwd packages/react-native featureflags",
     "lint-ci": "./.github/workflow-scripts/analyze_code.sh && yarn shellcheck",
     "lint-java": "node ./scripts/lint-java.js",
+    "lint-kotlin-check": "./gradlew :packages:react-native:ReactAndroid:ktfmtCheck",
+    "lint-kotlin": "./gradlew :packages:react-native:ReactAndroid:ktfmtFormat",
```

Comment:
We probably want a top level task inside the `build.gradle.kts` file that invokes several other tasks.

So you will invoke `./gradlew ktfmtCheck` and it will format both ReactAndroid and the Gradle plugin, etc.

Reply:
I moved it to the top level build.gradle.kts and configured it to also check other projects other than `ReactAndroid`. However, I'm not sure whether my new approach is the correct one and whether there is something else I should include.

Tested mostly with the packages `react-native/ReactAndroid` and `gradle-plugin`